webapp/src/components/ImageEditor/ImageEditor.tsx

```diff
@@ -35,6 +35,7 @@ export interface ImageEditorProps {
     onTileEditorOpenClose?: (open: boolean) => void;
     nested?: boolean;
     lightMode?: boolean;
+    hideDoneButton?: boolean;
```

Comment:
not for this pr, but next time we add a prop like this we should gather all the switches into a property bag. would save us a lot of passing props down in the image editor and music editor